src/applications/vaos/new-appointment/components/VAFacilityPage/FacilitiesRadioWidget.jsx

```diff
@@ -21,6 +21,7 @@ export default function FacilitiesRadioWidget({
   value,
   onChange,
   formContext,
+  required,
```

Comment:
Don't we need to add this for the covid facilities radio widget as well?

Reply:
Yes. Thanks